akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -762,6 +772,8 @@ class HttpExt private[http] (private val config: Config)(implicit val system: Ex
 
     webSocketClientLayer(request, settings, log)
       .join(_outgoingTlsConnectionLayer(host, port, settings.withLocalAddressOverride(localAddress), ctx, log))
+      // also added webSocketClientLayer but we want to make sure it covers the whole stack
+      .addAttributes(cancellationStrategyAttributeForDelay(settings.streamCancellationDelay))
```

Comment:
Same here, `(blueprint.join(tls)).addAttr(delay)` should be enough, or is the clientLayer used separately somewhere?